repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/checkoutrepository/CheckoutRepository.java

```diff
@@ -98,6 +104,8 @@ protected StepStatus businessExecute() {
                     List<String> paths;
                     if (checkoutType == CheckoutType.CHECKOUT_BUGGY_BUILD_SOURCE_CODE) {
                         paths = this.getPaths(this.getInspector().getJobStatus().getRepairSourceDir(), git);
+                    } else if (checkoutType == CheckoutType.CHECKOUT_BUGGY_BUILD_TEST_CODE) {
+                        paths = this.getPaths(this.getInspector().getJobStatus().getTestDir(), git);
```

Comment:
`getTestDir()` might be null in some cases (in multi-module projects for example). Then you might have at worst a NPE, and even without the NPE I'm not sure this will work! You should check this code by testing it with a multi-module project!

Reply:
See #518

Reply:
I will test it with a multi-module project.

Reply:
> You should check this code by testing it with a multi-module project!

I tested with a multi-module project and I got a perfect branch: https://github.com/fermadeiral/bears-collection/tree/fermadeiral-test-repairnator-bears-386337343-20180531-163325

> `getTestDir()` might be null in some cases (in multi-module projects for example). Then you might have at worst a NPE

To make sure I won't have a NPE, I made `ComputeTestDir` a blocking step in `ProjectInspector4Bears` when passing/passing.

Reply:
OK so you did that in purpose: note that we might loose a lot of builds because of that. OK?

Reply:
I am aware that we might lose builds, if it's a lot I still don't know. But currently I have no better solution.